<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>individualStudent</class>
 <widget class="QMainWindow" name="individualStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="individualStudentId"/>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="individual_logout_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #D7BDE2;
    color: black;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #AF7AC5;
}
QPushButton:pressed {
    background-color: #884EA0;
}
</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="individual_home_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #85C1E9;
    color: black;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #5DADE2;
}
QPushButton:pressed {
    background-color: #3498DB;
}
</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="individual_view_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #F1948A;
    color: black;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #EC7063;
}
QPushButton:pressed {
    background-color: #CB4335;
}
</string>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" rowspan="2" colspan="3">
     <widget class="QWidget" name="barchartStudent" native="true"/>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="individual_percentage">
      <property name="text">
       <string>Percentage - </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Student Id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Average :- </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
